util/conn.go

```diff
@@ -62,7 +62,7 @@ func ESConn(env Environment) (*elastic.Client, error) {
 	var op []elastic.ClientOptionFunc
 	op = append(op, elastic.SetHttpClient(&http.Client{Timeout: 30 * time.Second}))
 	op = append(op, elastic.SetURL(env.EnvString("ESURL")))
-	op = append(op, elastic.SetSniff(true))
+	op = append(op, elastic.SetSniff(!env.IsDebug()))
```

Comment:
We do not accept dedicated env configuration to launch in only the docker.
Please consider to it can be run on multiple environments.

Reply:
Sorry.
I was thinking of creating a new environment variable to disable Sniff if this is not accepted.
I will close this Pull Request.